content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -444,6 +444,8 @@ either be a snapshot file from a previous backup operation, or from a remaining
    ```shell
    export ETCDCTL_API=3
    etcdctl --data-dir <data-dir-location> snapshot restore snapshot.db
+or 
+   etcdctl snapshot restore --data-dir <data-dir-location> snapshot.db
```

Comment:
Thanks for opening this PR. 
This command is the same thing as the one in Line 446. The difference is just the arrangement.